src/components/events/EventCard.jsx

```diff
@@ -29,24 +59,35 @@ export default function EventCard({ event }) {
           <b>Venue:</b> {event.venue}
         </p>
       </div>
-      {token && (
-        <div className="going-buttons">
-          {attending ? (
-            <>
-              <button className="selected-button">Going</button>
-              <button className="not-selected-button">Not going</button>
-            </>
-          ) : (
-            <>
-              <button className="not-selected-button">Going</button>
-              <button className="selected-button">Not going</button>
-            </>
-          )}
-          <button className="info-button">More info</button>
-        </div>
-      )}
-
-      {/* <Link to={`/events/${event.id}`}>More info</Link> */}
+      <div className="event-buttons">
+        {token && (
```

Comment:
Sorry, one more thing I just realized:

We might be able to use `user &&` here instead of `token &&` - it could be cleaner for the code to reduce the selector imports

Reply:
You are right  🤔

Reply:
I just fixed that, now is using user and deleted the token lines.